internal/sirius/create_person.go

```diff
@@ -0,0 +1,55 @@
+package sirius
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+)
+
+type PersonType string
+
+const (
+	PersonTypeDonor = PersonType("donor")
+)
+
+func (c *Client) CreatePerson(ctx Context, personType PersonType, person Person) (Person, error) {
```

Comment:
I think it may have been better to have this as CreateDonor for now, and decide when we do the next one whether to combine or not